README.markdown

```diff
@@ -354,6 +354,19 @@ queues created on the fly, you can use a splat:
 Queues will be processed in alphabetical order.
 
 
+### Running All Queues Except for Some
+
+If you want your workers to work off of all queues except for some,
+you can use negation:
+
+    $ QUEUE=*,!low rake resque:work
+
+Negated globs also work. The following will instruct workers to work
+off of all queues except those beginning with `file_`:
+
+    $ QUEUE=*,!file_*
+
+
```

Comment:
Worth nothing that order doesn't matter for skipped queues?

Reply:
Good call. I updated accordingly.